### PR TITLE
Ambassador 0.52.0

### PR DESCRIPTION
**Pull request by kflynn**
Upgrade Ambassador to 0.52.0, and delete the duplicate `type` entry in `service.yaml`.

- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped

### CODE REVIEW

File: stable/ambassador/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: 0.51.2
+appVersion: 0.52.0
```
Comment (Flydiverny) [inline stable/ambassador/Chart.yaml:2]: update `image.tag` in values as well :) thats the important one. I know there used to be a template which selected the app version if no image tag was set but I felt this was a bit to magic.

**Review by Flydiverny — CHANGES_REQUESTED** [review]

File: stable/ambassador/Chart.yaml (hunk 1 above)
Comment (kflynn) [reply stable/ambassador/Chart.yaml:2]: Whoops. Done, and in the README too.

**Pull request merged**